components/constants/pref_names.h

```diff
@@ -58,16 +58,20 @@ extern const char kNewTabPageShowsOptions[];
 extern const char kNewTabPageCustomBackgroundEnabled[];
 extern const char kBraveTodayIntroDismissed[];
 extern const char kAlwaysShowBookmarkBarOnNTP[];
-extern const char kAutocompleteEnabled[];
-extern const char kTopSiteSuggestionsEnabled[];
-extern const char kBraveSuggestedSiteSuggestionsEnabled[];
 extern const char kBraveDarkMode[];
 extern const char kOtherBookmarksMigrated[];
 extern const char kBraveShieldsSettingsVersion[];
 extern const char kBinanceAccessToken[];
 extern const char kBinanceRefreshToken[];
 extern const char kDefaultBrowserPromptEnabled[];
 
+// Omnibox Configuration
+extern const char kAutocompleteEnabled[];
+extern const char kTopSiteSuggestionsEnabled[];
+extern const char kBraveSuggestedSiteSuggestionsEnabled[];
+extern const char kHistorySuggestionsEnabled[];
```

Comment:
We should not add new constants here and existing ones should be cleaned up - https://bravesoftware.slack.com/archives/C7VLGSR55/p1654103087023839

I think we can add `pref_names.h` to `//brave/components/omnibox/browser/` for above.

Reply:
Hahaha I actually had them there originally, but moved them here when I rebased and noticed the prefs had moved!

Reply:
Moved back,